reporters/SponsorReporter.js

```diff
@@ -16,12 +16,21 @@ class SponsorReporter {
 
     this.formatDecimalString = createFormatFunction(this.web3, 2, 4);
 
+    this.lastUpdateTimestamp = 0;
+    this.updateThresholdSeconds = 60;
```

Comment:
We can allow the user to pass this in by construction but because the SponsorReporter is not used outside of this module I believe it to be overoptimization.

Reply:
+1